app/views/courses/show.html.erb

```diff
@@ -48,6 +55,9 @@
           <br>
           Course status:
           <%= @course.status&.humanize %>
+          <br>
```

Comment:
<br>
          Required attendance rate:
          <%= @course.attendance_rate %>%
to
<p>Required attendance rate: <%= @course.attendance_rate %>% </p>